### PR TITLE
Some small changes to ui

### PR DESCRIPTION
**Pull request by jonhartzler**
fix some of the easy to fix warnings.  and added some there things to the partners pages and the way that external links are linked to.

### CODE REVIEW

File: src/components/StaticAssets/Footer.jsx [hunk 1]
```diff
@@ -13,8 +13,8 @@ class Footer extends Component {
             { this.props.showBackToSearchButton ? (<BackToSearchButton/>) : '' }
           </div>
           <div className="col-sm-6 right">
-            <a href="http://humanrights.nd.edu/"><img src="/resources/images/cchr.png" className="cchr" /></a>
-            <a href="https://library.nd.edu/cds"><img src="/resources/images/undhl-cds.png" className="undhl" /></a>
+            <a href="http://humanrights.nd.edu/" target="_blank" ><img src="/resources/images/cchr.png" alt="The Center For Civil and Human Rights" className="cchr" /></a>
+            <a href="https://library.nd.edu/cds" target="_blank" ><img src="/resources/images/undhl-cds.png" alt="Hesburgh Libriary Center for Digital Scholarship University of Notre Dame" className="undhl" /></a>
```
Comment (jgondron) [inline src/components/StaticAssets/Footer.jsx]: Check spelling

File: src/components/StaticPages/Partners.jsx [hunk 2]
```diff
@@ -20,49 +20,50 @@ var Partners = React.createClass({
         		<table>
     			  <tbody>
     			    <tr>
-    			      <td><img src="/resources/images/partners/ndls.png" /></td>
-    			      <td><img src="/resources/images/partners/kiis.png" /></td>
+    			      <td><img alt="University of Notre Dame Law School" src="/resources/images/partners/ndls.png" /></td>
+    			      <td><img alt="Kellog Institute for International Studies" src="/resources/images/partners/kiis.png" /></td>
```
Comment (jgondron) [inline src/components/StaticPages/Partners.jsx]: Kellogg?

Comment (jonhartzler) [inline src/components/StaticPages/Partners.jsx]: yup

**Comment by jgondron** [comment]
:+1:

**Pull request merged**